tests/integration/pure_cairo_vm/cheatcodes/deploy/deploy_test.cairo

```diff
@@ -6,10 +6,10 @@ func test_deploying_pipeline_with_path(){
     local deployed_hash;
 
     %{
-        declared = declare("./src/main.cairo")
+        declared = declare("./src/main.cairo").unwrap()
```

Comment:
It's a bit confusing that sometimes we use `unwrap()` and sometimes `ok`... Why don't we always use `unwrap()`?

Reply:
I added `unwrap` later, and I didn't change older code.